core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/CachableBlockFile.java

```diff
@@ -147,7 +147,7 @@ public long getStartPos() throws IOException {
   public static class Reader implements BlockFileReader {
     private final RateLimiter readLimiter;
     private BCFile.Reader _bc;
-    private String fileName = "not_available";
```

Comment:
OK I see why you made the changes but this old fileName does make me curious as to why "not_available" was chosen... any idea?